binary_and_decimal.rb

```diff
@@ -13,9 +27,20 @@ def binary_to_decimal
 #       You may use array indexing to retrieve one element at a time,
 #       compare individual elements with each other and
 #       you may retrieve the length of an array.
-def array_equals(array1, array2)
-  puts "Not implemented"
-  return true
+def array_equals(first, second)
+  total = 0
+  if first.length == second.length
+    first.each_with_index do |integer, i|
+      if integer == second[i]
```

Comment:
Minor suggestion for improvement. This loop will run length number of times even if all the elements are different. This may be significant run time when the arrays are really long. You could improve on this by changing the conditional check to `if integer != second[i]` and returning false in that case.